tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -403,6 +403,98 @@ public function testUpdateWithSameColumnInDataAndIdentifiers()
         );
     }
 
+    /**
+     * @group DBAL-2688
+     */
+    public function testUpdateWithIsNull()
+    {
+        $driverMock = $this->createMock('Doctrine\DBAL\Driver');
+
+        $driverMock->expects($this->any())
+            ->method('connect')
+            ->will($this->returnValue(new DriverConnectionMock()));
+
+        $conn = $this->getMockBuilder('Doctrine\DBAL\Connection')
```

Comment:
Can you extract this to a method (you can also encapsulate `$driverMock` there) so that it can be reused on `delete()`?